iteration/iteration.go

```diff
@@ -187,14 +187,19 @@ func (m *GormIterationRepository) Root(ctx context.Context, spaceID uuid.UUID) (
 	var itr Iteration
 
 	tx := m.db.Where("space_id = ? and path = ?", spaceID, "").First(&itr)
-	if tx.Error != nil && tx.Error != gorm.ErrRecordNotFound {
+	if tx.Error != nil {
```

Comment:
Minor - to eliminate nesting we could do if `tx.Error == nil { return &itr, nil}` instead

Reply:
Thanks @tinakurian. Please see https://github.com/fabric8-services/fabric8-wit/pull/2126/commits/99a6c67a44651f6a150f5d71785ea38d5d611c1a